shared/base/view.js

```diff
@@ -195,7 +195,7 @@ module.exports = BaseView = Backbone.View.extend({
    * Get the HTML for the view, including the wrapper element.
    */
   getHtml: function() {
-    var html = this.getInnerHtml(),
+    var html = this.options.lazy ? '' : this.getInnerHtml(),
```

Comment:
Okay, lemme try that out, and I can close this and open a PR to rendr-handlebars if it was as intended.